pixi.toml

```diff
@@ -0,0 +1,50 @@
+[project]
+authors = ["IRIS-HEP"]
+channels = ["conda-forge"]
+description = "Add a short description here"
+name = "analysis-grand-challenge"
+platforms = ["linux-64"]
```

Comment:
@alexander-held Is it useful for local testing to also support `osx-64` and `osx-arm64`? Or do you do all your development on a coffea-casa instance and so just Linux support is sufficient?

Reply:
I also run locally so indeed that does seem useful to add!